docs/containerd.md

```diff
@@ -30,4 +30,13 @@ containerd_registries:
     - "https://registry-1.docker.io"
 ```
 
+`containerd_registries` is ignored for pulling images when `image_command_tool=crictl`
+(the default for `container_manager=containerd`). Use `crictl` instead, it supports
+`containerd_registries` but lacks proper mult-arch support (see
```

Comment:
s/mult-arch/multi-arch/ according to https://github.com/kubernetes-sigs/kubespray/issues/8375#issuecomment-1005837307

Reply:
Oups. Thanks. Fixed.